src/Sylius/Bundle/ApiBundle/Attribute/PaymentRequestActionAware.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Attribute;
+
+#[\Attribute(\Attribute::TARGET_CLASS)]
+final class PaymentRequestActionAware
```

Comment:
Add to these newly created PaymentRequest-related classes an experimental tag, we will remove it at once when we stabilize this feature 👌🏻